packages/react-native/Libraries/Text/Text/RCTTextShadowView.mm

```diff
@@ -85,6 +85,9 @@ - (void)uiManagerWillPerformMounting
 
   NSNumber *tag = self.reactTag;
   NSMutableArray<NSNumber *> *descendantViewTags = [NSMutableArray new];
+  NSMutableArray<NSNumber *> *virtualSubviewTags = [NSMutableArray new];
+
+#if !TARGET_OS_OSX // [macOS]
```

Comment:
why do we not want the descendantViewTags enumerated on MacOS?

Reply:
I don't think we need to #def the changes you're making unless you're running into a compatibility issue with building for other platforms.

Reply:
We DO enumerate descendantViewTags on macOS, as seen in [lines 105–109 of the same file](https://github.com/microsoft/react-native-macos/pull/2143/files/ee372a91cee9976b75158240e0da04b43b375bbe#diff-96ef859d59f8ab7409088d646fe933ec3b2e3f0d631bdb672f3f6fd6c540f412R105-R109).

Or are you recommending that we call `enumerateAttribute:inRange:options:usingBlock:` twice (once per relevant attribute) on macOS instead of a single `enumerateAttributesInRange:options:usingBlock:`?

Reply:
Upon further reflection, it might be better to enumerate both on both platforms anyway. Mouse events (or something similar) could be relevant on iPad as well when you have a pointer involved.

Reply:
Bad comment on my part trying to review quickly Friday evening, apologies -- I either misread it as an `#if - #else - #endif` or misspoke looking to understand why you'd limited your new functionality to TARGET_OS_OSX